src/exchanges/cryptoswap/basics.js

```diff
@@ -0,0 +1,22 @@
CXv9hZlaRYlJN4/dan4d/IrePi8PTp/tCCKE4/uubjmTyDwmpaR/9qc1DTf2X5Jqqrvno/THAkN4t4VJco5AzhJinEu5Sku7fVOr8erFDKtqv5z+fc6BRADJPMmPWKyWcbxCNqjS51gL0u9bv/9ftG9bM3NT/3tLxpf1jks6EDCHYsIpWFHl1QGQJhCFe0txen7v3kDxnrbDxkitbi93TsQSc9PewwL5nf8jevuL99oZo4ImyXI4jXHBpDpXqe390d9H+FLrVqZKStrMvdauYTY0aIscAEc4yCTA0Tzq98JhN77+rUjztvDzOoMQZ0DyI0fT3LoDukdQRI5AbRcvlY3/mXQFb/vY13D9MytBRt3pKXVmJmEh7n47+jYWrA7BksC5QGHBpl2BYBoTNV+9Pq8Y765eexRgoe+/30xpaSwRPn50bdPKwBUnUHjZQavnPYgpZhICAk7tqSDW9cCA2Z2H7XknwLJ2DXv3ytS8i4hCC/HJSEsQ9XZoDc7jTgvRXGqa+OK41qTe877sjcsmNqQyMIUQkkvwj2t5YIbwRdZ13V9L9O0xbqO6d/fPIKv26tDykZv+2pSErPJowhj1wmSeCnnQS83xlGAMP0vRdHv/qr38C21aaiBs9PMUqOEjwilx9y+xgPZIb/XP2pOrzGLavP5vO4xK9+502Qy9l6xwf4bExeno5uiHvI+M3HAWG9tt6G2669zM0i5wa796JXEsOiYZZSTPFQASPlXgRBwDYB8Xdd3+AqKp7sKjqxe/8g7uSd+eds+MjAi6cpW10gO+wOiJHYkQf++kfJb0XjGplzXcz3FRR0WD3j6jDfhNRnwwd+/FUUttv2EkPAKSy3GdG2PElCHzO45ZnVN6qtJ2RFL3rzL6gibQIUgS0kFgOzPL/Z0+OCG8z/auDr9rbMA4Z3nNJ7uyxPWE1G4+MSvfbkf8jgDEOIE0/dQEDcoLGBoRgBO3masq48DxPE/82XVLq/T03PxgLGZ1TH82Soz+LtJ7WiYeTUF7ME6WdDHytgRb6mz/8ENnvXVZaCsTpt4tHK9+Nj+giRM4E6HinZkjLmKjhRePuO2Zw12+rpw1WmA8AG6ampaXERK4kRCaT9Cgi+RcZhY1vMTX/pjX8PTAqICYJiiKdpSxa08/82/xvAgyX9sg8zbypdatH7MWFGSngnuR4IAKXMzOwMe77OBvLz5H9/36t/e+9w/9dF6Ec0b3CdZzE+VsT4eO/BgrFQLqI9N6DLqfOQHOGOs1nmAcMv0nvtegiKo98sW89OEUiPx6KwAhCsGuNxvMlWf/c2AYxvyMx6NM7xx4LfvJUgW4S0i0AGEMOG42Y4xr6YoS/zF7nc8n29ecyYu5v5p/eW4qzr1EWX5Qckk9eTeqmNOCaPRAV3XJjll4cXplw89rUb6GXbxnN32PwEQbt0bJ402aXFNm4l286dUpBdzauaKs0X5MqpaMwgn0NbUvbpGH8r+a+eGdWMnnvPN6JBFb8Ywu30ppWh78v6IcSdEPtP0Xw78eeDhJWPe4MvA6sx0ZMiiV1tYIiJfFQTaBQSRJ9mN81lr+heCp2TYuOufPi2N0Tl7y//Gg/9nAFJuo+T+/S3N+l6WaouNHEOokEwpsZaR1J40qxy3/KJcaUb3cWEq3eX91LVp94Tlr71XLY2SvzE+Nbq1a9pAc4uuV2wkAm1leO/KroozJie50ALKCl+pf6bfXbojPjmxxFPiUnxKgJklmciRJtlf6I5kAm1ji7CPooRyx4R44oxb9nXRdVX7afOnK/utmjKvVpjua2SAWij8PweQchtePe7JJHvTpFtkq6UTY2hBgGQq0GgQGBTWxueVMR06K+KiPYRgj+YPbCrYufeLNU9P+ke8VGcy3vd+9EqiNSVyBqXC9UHa0uP3XMdmSMJAmBdMz1Y8/iOEUj/RdbNktdSjFHFgsJR7/ypdkjJjWbVg57I1Q5akfXLeUC6dic1Od8//LEDKjdJ00mhTg5RmTSSV1JetYozi8TmYpkqggmq2WlyqohUoPn9m6cGc/WueujBiigYumdjQZJKfpaJwPwgxIn8r0g6dsH84dshZYco5WqYSpwZjrISp2rTsjIzXP73n9Qv6pLwqQP3PA+QkA6WBYnt/gtbz2IVAjXmqAR60cJzdr+j3hMdHce7io0uk4/ZYJ3jyjr4Mxx+sHuf1g64XlRa6HmHwfDm9X1qd2pBXZssQQKr6hFzgf7/n45cusSfHzBQE2hqEnBAWE+xcxdmi/Bcn/I7xdH9dx87c3zPu+HTUK3+L6f1CMmkIIBfSaJ1hW/tMGR5Zr9lFdwqicBMVxCtBwU/eg7LnpwcIFxDJh6av83u8X+TvyFgw7zzw2J2hGc7othBAzshsF+RNZMhXryQQh72TqqrtREnqIpnkllRAdLlwYdnLwIntcpmq7wh4/MstdsufpUVFm2be/NxJYTgXpBVq2OgQQGposLpQnJ8J1W/UwG4Kk81F+3NtGtWOvgcSNelybJTH5Cv1/pp3wLl56IXDA1wbYxMCSG1YNVRnnbFACCB1ZihDHakNC4QAUhtWDdVZZywQAkidGcpQR2rDAiGA1IZVQ3XWGQuEAFJnhjLUkdqwQAggtWHVUJ11xgIhgNSZoQx1pDYsEAJIbVg1VGedsUAIIHVmKEMdqQ0LhABSG1YN1VlnLBACSJ0ZylBHasMCIYDUhlVDddYZC4QAUmeGMtSR2rBACCC1YdVQnXXGAiGA1JmhDHWkNiwQAkhtWDVUZ52xQAggdWYoQx2pDQuEAFIbVg3VWWcsEAJInRnKUEdqwwIhgNSGVUN11hkLhABSZ4Yy1JHasEAIILVh1VCddcYCIYDUmaEMdaQ2LBACSG1YNVRnnbFACCB1ZihDHakNC4QAUhtWDdVZZywQAkidGcpQR2rDAiGA1IZVQ3XWGQuEAFJnhjLUkdqwQAggtWHVUJ11xgIhgNSZoQx1pDYs8P+PxjnVoUB1UgAAAABJRU5ErkJggg==" /',
```

Comment:
Looks malformatted.

I think you will need to remove `" /` at the end of this string.

Reply:
Also make sure you pipe the png through https://kraken.io/web-interface before you convert it to base64

![Screenshot 2022-06-04 at 18 33 07](https://user-images.githubusercontent.com/851393/172016561-baf5e778-01dd-448e-adf0-7b7927974de5.png)

Reply:
I also find the logo hard to recognize:

![Screenshot 2022-06-04 at 18 39](https://user-images.githubusercontent.com/851393/172017085-b399cd8f-a72e-4aa5-af4a-6f33c053174f.png)

Do you maybe have a simpler version of the logo (with less elements)?